qa/rpc-tests/test_framework/util.py

```diff
@@ -36,10 +37,15 @@ def sync_blocks(rpc_connections, wait=1):
     """
     Wait until everybody has the same block count
     """
+    j=0
     while True:
+        j+=1
         counts = [ x.getblockcount() for x in rpc_connections ]
+        print "   Syncing...Node Block Counts ==> " + str(counts) + ' Elapsed Time (s): ' + str(j)
```

Comment:
The ident was intentional.  When you run the script it makes it easier 
to read the output lines and see what part of the script is running.  But
as I said, I think the whole logging of messages needs to be updated and 
standardized.

On 16/08/2015 5:24 AM, MarcoFalke wrote:
>
> In qa/rpc-tests/test_framework/util.py 
> <https://github.com/bitcoin/bitcoin/pull/6548#discussion_r37145288>:
>
> >          counts = [ x.getblockcount() for x in rpc_connections ]
> > +        print "   Syncing...Node Block Counts ==> " + str(counts) + ' Elapsed Time (s): ' + str(j)
>
> Should probably say something like |print "sync_blocks: " + str(counts 
> ...|?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/bitcoin/bitcoin/pull/6548/files#r37145288>.
>